src/main/java/de/tum/in/www1/artemis/domain/enumeration/ProgrammingLanguage.java

```diff
@@ -10,7 +10,9 @@
 
 public enum ProgrammingLanguage {
 
-    JAVA("java"), PYTHON("py"), C("c", "cpp"), HASKELL("hs"), KOTLIN("kt"), VHDL("vhd"), ASSEMBLER("asm"), SWIFT("swift"), OCAML("ml"), EMPTY("");
+    EMPTY(""), JAVA("java"), PYTHON("py"), C("c", "cpp"), HASKELL("hs"), KOTLIN("kt"), VHDL("vhd"), ASSEMBLER("asm"), SWIFT("swift"), OCAML("ml"), JAVASCRIPT("js"), C_SHARP("cs"),
```

Comment:
can we put each enum value on a new line? either customize the formatter or turn it off here (see e.g. SecurityConfiguration)

Reply:
I tried the `alignment_for_enum_constants` config in `artemis-spotless-style.xml` but the concrete values are not documented. Some values I found online had an effect on many other unrelated code parts.

I'll try disabling the formatter with the comment, thank you.